### PR TITLE
docs: updates list of countries in country-map-tools.mdx

### PR DESCRIPTION
**Pull request by jbat**
Updated current list of countries from the source list in
https://github.com/apache/superset/blob/master/superset-frontend/plugins/legacy-plugin-chart-country-map/src/countries.ts#L117

### CODE REVIEW

**Review by rusackas — APPROVED** [review]
Thanks! I'm hoping to make this an automatic update at some point, but really appreciate you bringing this current and adding the source reference!

File: docs/docs/miscellaneous/country-map-tools.mdx [hunk 1]
```diff
@@ -16,39 +16,107 @@ code is less ambiguous and is unique to all regions in the world.
 
 ## Included Maps
 
+The current list of countries can be found in the src
```
Comment (rusackas) [inline docs/docs/miscellaneous/country-map-tools.mdx:19]:
```suggestion
The current list of countries can be found in the src
```
Linting is failing on some trailing whitespace.

File: docs/docs/miscellaneous/country-map-tools.mdx [hunk 2]
```diff
@@ -16,39 +16,107 @@ code is less ambiguous and is unique to all regions in the world.
...
+-  Timorleste
+-  Turkey
+-  Turkey Regions
+-  Turkmenistan
+-  Uganda
+-  Uk
+-  Ukraine
+-  United Arab Emirates
+-  Uruguay
+-  Usa
```
Comment (rusackas) [inline docs/docs/miscellaneous/country-map-tools.mdx:115]:
```suggestion
-  USA
```

**Comment by mistercrunch** [comment]
Thanks for the PR!

**Pull request merged**